maps_dashboards/public/components/map_top_nav/get_top_nav_config.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { i18n } from '@osd/i18n';
+import { TopNavMenuData } from '../../../../../src/plugins/navigation/public';
+import {
+  OnSaveProps,
+  SavedObjectSaveModalOrigin,
+  showSaveModal,
+} from '../../../../../src/plugins/saved_objects/public';
+import { MapServices } from '../../types';
+
+export const getTopNavConfig = (services: MapServices) => {
+  const {
+    notifications: { toasts },
+    i18n: { Context: I18nContext },
+    savedObjects: { client: savedObjectsClient },
+  } = services;
+
+  const topNavConfig: TopNavMenuData[] = [
+    {
+      iconType: 'save',
+      emphasize: true,
+      id: 'save',
+      label: i18n.translate('maps.topNav.saveMapButtonLabel', {
+        defaultMessage: `Save`,
```

Comment:
Can also consider adding logics here to enable/disable the save button. Currently in visBuilder, if no change is being made, or visBuilder is invalid, we disable the save button.

Reply:
Same idea, I plan to include this logic in another PR. Thanks!